VRCFaceTracking/Params/Expressions/UnifiedSimpleExpressions.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace VRCFaceTracking.Params
+{
+    /// <summary> 
+    /// Represents the type of transformed and simplified Shape data that is converted from the base UnifiedExpression data in the form of enumerated shapes.
+    /// </summary>
+    /// <remarks>
+    /// These shapes have a basis on general common expressions from different tracking interfaces and standards 
+    /// and are directly transformed from Unified Expressions data.
+    /// </remarks>
+    public enum UnifiedSimpleExpressions
+    {
+        BrowDownRight, // Lowers and pinches the right eyebrow.
+        BrowDownLeft, // Lowers and pinches the left eyebrow.
+        MouthSmileRight, // Moves the right corner of the lip into a smile expression.
+        MouthSmileLeft, // Moves the left corner of the lip into a smile expression.
+        MouthSadRight, // Moves the right corner of the lip into a sad expression.
+        MouthSadLeft, // Moves the left corner of the lip into a sad expression.
+        MouthUpperUpRight, // Raises the entire upper right of the lips.
+        MouthUpperUpLeft, // Raises the entire upper left of the lips.
+    }
```

Comment:
I added these to be more recognizable and simplified expressions derived from UnifiedExpressions base parameters. These are also intended to be used as both base parameters and used in merged params. They are different from merged params as they are more recognizable as base params from other standards such as AR52 and SRanipal for example. They also have a conversion map as well in the same .cs file. 

I'm not entirely sure the best approach for this class, or if it even should be separated from the UE file, but I envisioned that as the base UE parameters eventually get segmented out (if the standard continues to grow) that the then child of said parameters would eventually get fed into this class and dynamically iterate the standard whilst keeping backwards compatibility, hence the name.